lib/utilities/aclUtils.js

```diff
@@ -233,4 +234,72 @@ aclUtils.sortHeaderGrants =
         return addACLParams;
     };
 
+/**
+ * convertToXml - Converts the `grantInfo` object (defined in `objectGetACL()`)
+ * to an XML DOM string
+ * @param {object} grantInfo - The `grantInfo` object defined in
+ * `objectGetACL()`
+ * @return {string} xml.join('') - The XML DOM string
+ */
+aclUtils.convertToXml = grantInfo => {
+    const { grants, ownerInfo } = grantInfo;
+    const xml = [];
+
+    // Escape any special XML characters
+    for (const key in grants) {
```

Comment:
Addressed in [this commit](https://github.com/scality/S3/pull/289/commits/14c48295550a9b8598514a5d5440efdbe17421d6).